transport/nio/pom.xml

```diff
@@ -61,6 +61,59 @@
             <artifactId>jmock-legacy</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.kaazing</groupId>
+            <artifactId>specification.tcp</artifactId>
+            <version>${project.version}</version>
```

Comment:
This version is wrong.  It should be the latest k3po version, which is most likely inherited, so either remove the version or update it to the latest if it is not inherited